client/src/components/Search/Search.css

```diff
@@ -0,0 +1,20 @@
+.loader {
```

Comment:
we have this class and it's CSS in the `Spinner` component, so no need to have it here

Reply:
damn it that was the mystery of why it was working, I should open less tabs at once lol